app/views/posts/index.html.erb

```diff
@@ -1,4 +1,19 @@
-<h1>List of posts of a user</h1>
-<h2>app/views/post/index.html.erb</h2>
-<h2><%= __FILE__ %></h2>
-<h2><%= File.basename(__FILE__)%></h2>
+<%= render 'users/user_card', user: @posts[0].author %>
+<ul>
+  <% @posts.all.each_with_index do |post, i| %>
+  <% comments = Post.recent_comments(post.title) %>
+    <li><%= link_to render('post_card', post: post, i: i), user_post_path(@posts[0].author, post) %></li>
+    <div class="recent_comments border">
+    <h3 class="comments_tit">5 Most Recent Comments:</h3>
+    <%comments.each do |comment| %>
```

Comment:
- [ ] As the previous reviewer mentioned kindly make sure that the last 5 comments are shown on this page, I see that you included some images that show the comment seem to work however it's not the case for the code you pushed to this PR

![Screenshot_20230413_143529](https://user-images.githubusercontent.com/25227022/231795107-e0e97257-692b-4c3e-9676-59b528acb5e3.png)

Reply:
@ivanmvh, for some reason it's working now, It's my fault sorry, you can request a new review and the reviewer will approve your PR

Reply:
Thank you. I will ask for a new review.